packages/terra-clinical-result/src/result-name-header-cell/ResultNameHeaderCell.jsx

```diff
@@ -17,6 +18,14 @@ const propTypes = {
    * Content to be displayed as the supporting Unit of Measure row label.
    */
   unit: PropTypes.string,
+  /**
+   * Full name of the Result Name for accessibility toolings.
+   */
+  fullResultName: PropTypes.string,
+  /**
+   * Full name of the unit for accessibility toolings.
```

Comment:
Could you include "unit of measure" similar to the one above and with an example?